test/functional/helpers/edgeInfo/domain.js

```diff
@@ -12,10 +12,6 @@ const domains = {
 };
 
 export default {
-  getThirdParty() {
-    return domains[env].third;
-  },
-  getFirstParty() {
-    return domains[env].first;
-  }
+  thirdParty: domains[env].third,
+  firstParty: domains[env].first
```

Comment:
Why don't you just rename `third` and `first` to `thirdParty` and `firstParty` and then do `export default domains[env];`?